runner.py

```diff
@@ -8,7 +8,18 @@
 
     is_dummy = False
     args = ' '.join(sys.argv[1:])
+    if 'subdir' in config:
+        if not 'suite' in config:
+            print 'Test subdirectory present in config without specific test suite to run.'
+            sys.exit(1)
+        subdir = config['subdir']
+        suite = config['suite']
+    else:
+        subdir = None
+        suite = None
```

Comment:
It's nice to not include the keys in the config file when no specific suite or subdirectory is desired, and it's not clear to me if there's an API that returns default values for nonexistent keys.